samples/attested_tls/CMakeLists.txt

```diff
@@ -3,6 +3,19 @@
 
 cmake_minimum_required(VERSION 3.11)
 
+# If the CC environment variable has been specified or if the CMAKE_C_COMPILER
+# cmake variable has been passed to cmake, use the C compiler that has been
+# specified. Otherwise, prefer clang. Same for C++ compiler.
+# This must be done before the `project` command.
+if (UNIX)
```

Comment:
Do we need to do anything special for LVI?

Reply:
Currently we only enable the LVI mitigation build for a few samples (e.g., helloworld).

Reply:
Ok.